projects/plugins/protect/src/js/routes/scan/index.jsx

```diff
@@ -36,6 +38,9 @@ const ScanPage = () => {
 		currentScanStatus = 'active';
 	}
 
+	const hasHistory = history && history.threats.length;
+	const showResults = !! status.threats.length || hasHistory;
```

Comment:
I didn't check this specifically, but I think the `status.threat.length` check here might be off - I assume similar to `hasHistory` we want to show results  if `status && status.threats.length`?